habitat-baselines/habitat_baselines/rl/hrl/hl/fixed_policy.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List, Tuple
+
+import torch
+import yaml
+
+from habitat.config.default import get_full_config_path
+from habitat.tasks.rearrange.multi_task.rearrange_pddl import parse_func
+from habitat_baselines.common.logging import baselines_logger
+
+
+class FixedHighLevelPolicy:
```

Comment:
Doc string is needed when the policy is used, how it differs from other classes of HighLevelPolicies.